### ISSUE TITLE
No Applied Energistics 2 in JEI 

### ISSUE THREAD

**Issue by trechetican** [AllTheMods/ATM-5#239]
## Expected Behavior
seeing Everything from the Applied energistics 2 mod.

## Current Behavior
only see the facades from the AE2 mod

## Steps to Reproduce

1.Search for Applied energistics
2.only facades appear 
3.
4.

**Comment by bernie-g** [comment]
Can reproduce, several people in the discord are experiencing this too.

**Comment by Uginsecond** [comment]
Yeah, same problem - clean install, MultiMC - 5 pages of facades ...
